projects/workflows-creator/src/lib/services/bpmn/elements/tasks/create-jira-issue.task.ts

```diff
@@ -0,0 +1,44 @@
+import {Inject, Injectable} from '@angular/core';
+import {ENV} from '../../../../types';
+import {CreateStrategy, LinkStrategy} from '../../../../interfaces';
+import {ModdleElement} from '../../../../types/bpmn.types';
+import {UtilsService} from '../../../utils.service';
+import {CREATE_TASK_STRATEGY} from '../../strategies/create';
+import {LINK_BASIC_STRATEGY} from '../../strategies/link';
+import {ServiceTaskElement} from './service-task.task';
+import {ENV_TOKEN} from '../../../../token';
+
+@Injectable()
+export class CreateJiraIssue extends ServiceTaskElement {
```

Comment:
why are we hardcoding this task here ? Its not right. Talk to @akshatdubeysf on the right approach.